### PR TITLE
fix(RefinementList): fix showMore button styling

### PR DESCRIPTION
**Pull request by tkrugg**
This adds the CSS class `ais-RefinementList-showMore` to the show-more/show-less button
image

image

Closes https://github.com/algolia/angular-instantsearch/issues/377

### CODE REVIEW

**Comment by francoischalifour** [comment]
See story: https://deploy-preview-381--angular-instantsearch.netlify.com/examples/dev-novel/?selectedStory=RefinementList.panel%20header%20with%20showMore

**Comment by tkrugg** [comment]
yes I was hoping the snapshots would break but they didn't

File: src/refinement-list/__tests__/refinement-list.spec.ts [hunk 1]
```diff
@@ -105,21 +105,111 @@ describe('RefinementList', () => {
...
-    fixture.componentInstance.testedWidget.showMoreLimit = 10;
-    fixture.detectChanges();
+  describe('[show more]/[show less] button', () => {
+    const createFixture = (state, props) => {
+      const fixture = render(state);
+      Object.assign(fixture.componentInstance.testedWidget, props);
+      fixture.detectChanges();
+      return fixture;
+    };
+    it('should not show [show more] button when canToggleShowMore === false ', () => {
```
Comment (samouss) [inline src/refinement-list/__tests__/refinement-list.spec.ts]: The button should be visible when `canToggleShowMore` is `false` but with the disabled class (see [Vue](https://github.com/algolia/vue-instantsearch/blob/v2/src/components/RefinementList.vue#L76-L93)).

Comment (tkrugg) [reply src/refinement-list/__tests__/refinement-list.spec.ts]: ok that's another bug around the the showMore button.
https://github.com/algolia/angular-instantsearch/blob/bbe0e9631b4f09333097f5b01a7153889f031cb5/src/refinement-list/refinement-list.ts#L60

I understand now how that's not what `canToggleShowMore` was meant for. 
Wouldn't it be a breaking change to change this here? There may be users relying on this behaviour.

Comment (samouss) [reply src/refinement-list/__tests__/refinement-list.spec.ts]: Indeed, it can be solve separately then.

Comment (tkrugg) [reply src/refinement-list/__tests__/refinement-list.spec.ts]: ok, I filed an issue about this here to track it: https://github.com/algolia/angular-instantsearch/issues/391
PR is coming!

Comment (tkrugg) [reply src/refinement-list/__tests__/refinement-list.spec.ts]: I simplified the tests and in particular removed those specifying the inconsistent display of showmore button base on canToggleShowMore boolean.
I reintroduced them here where https://github.com/algolia/angular-instantsearch/pull/381/files with the correct behaviour

File: src/refinement-list/__tests__/refinement-list.spec.ts [hunk 2]
```diff
@@ -105,21 +105,52 @@ describe('RefinementList', () => {
...
+        {
+          limit: 5,
+          showMoreLimit: 10,
+        }
+      );
+      const showMoreBtn = fixture.debugElement.nativeElement.querySelector(
+        'button'
+      );
+      showMoreBtn.click();
+      expect(toggleShowMore).toHaveBeenCalled();
```
Comment (francoischalifour) [inline src/refinement-list/__tests__/refinement-list.spec.ts]: The assertion could probably be stricter:

```suggestion
      expect(toggleShowMore).toHaveBeenCalledTimes(1);
```

**Pull request merged**